src/Toggle/CreateMarketingInformationForToggle.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Volodymyr
+ * Date: 10/08/2015
+ * Time: 09:52
+ */
+
+namespace Clearbooks\Labs\Toggle;
+
+
+use Clearbooks\Labs\Toggle\Entity\CreateMarketingInformationRequest;
+use Clearbooks\Labs\Toggle\Gateway\MarketableToggleGateway;
+use Clearbooks\Labs\Toggle\UseCase\CreateMarketingInformationForToggle\MarketingInformationRequest;
+
+class CreateMarketingInformationForToggle
+{
+    /**
+     * @var MarketableToggleGateway
+     */
+    private $gateway;
+
+    /**
+     * CreateMarketingInformationForToggle constructor.
+     * @param MarketableToggleGateway $gateway
+     */
+    public function __construct( MarketableToggleGateway $gateway )
+    {
+        $this->gateway = $gateway;
+    }
+
+    /**
+     * @param MarketingInformationRequest $request
+     */
+    public function execute( MarketingInformationRequest $request )
+    {
+        $toggleId = $request->getToggleId();
+        $marketingInformation = [
+            'screenshot_urls' => $request->getImageLink(),
+            'discription_of_toggle' => $request->getDescriptionOfToggle(),
+            'discription_of_functionality' => $request->getDescriptionOfFunctionality(),
+            'discription_of_benefits_and_reasonForImplementation' => $request->getDescriptionOfReasonForImplementation(),
```

Comment:
You swap between underscores and camel case here, column names usually have underscores so stick with that. Its quite long though how about just `description_of_implementation_reason`

Also you have spelt 'Description' wrong :wink: